src/test/java/PalindromeCheckerTest.java

```diff
@@ -14,7 +15,7 @@ public void beforeEachTest() throws Exception {
 
     @Test
     public void palindromeCheckerReturnsTrueForSingleCharacterWords() {
-
+        //inline
```

Comment:
I think this would be easier to read if you just put the String in-line with the method call rather that putting it in a variable - as it is you have to look in two places, at the method call, then back up to see what value the variable is set to.